api/model.go

```diff
@@ -14,13 +14,17 @@ type (
 		Error string `json:"error"`
 	}
 	GetStatusResponse struct {
-		ID       string    `json:"id"`
-		Replicas []Replica `json:"Replicas,omitempty"`
+		ID      string   `json:"id"`
+		Replica *Replica `json:"Replicas,omitempty"`
```

Comment:
🤔 this should continue to be a slice right? A blob may have multiple replicas.

Reply:
That seems correct. I was a bit confused by the relationship between deals and pieces.  I think something needs to change [here](https://github.com/filecoin-project/motion/pull/153/files#diff-da5cbb9a08e25011c2c5c0ea433286ca1b844b015f8be8dca6da93ae7df99025R469-R482)

Does each deal map to a separate replica, or a separate piece, or either? So if a blob is spread across multiple pieces, are those pieces in separate deals? If so, then I need to know how to determine if separate deals from [`getFileDealsRes`](https://github.com/filecoin-project/motion/pull/153/files#diff-da5cbb9a08e25011c2c5c0ea433286ca1b844b015f8be8dca6da93ae7df99025R469) are for different replicas or for different pieces in the same replica.

Reply:
> something needs to change [here](https://github.com/filecoin-project/motion/pull/153/files#diff-da5cbb9a08e25011c2c5c0ea433286ca1b844b015f8be8dca6da93ae7df99025R469-R482)

Right, the work here is to set up stepping stones for further changes across the workflow.

> Does each deal map to a separate replica, or a separate piece, or either?

Mapping of replica:deal:piece is 1:1:1 for blobs that fit within a sector (32GiB currently). But as soon as blobs become larger, then the mapping of replica to deal looks something like this 1:1..*, and mapping of deal to piece (for the time being) continues to be 1:1.

Cc @xinaxu We should think about large blob support and mapping of Singularity primitives to Motion as part of work items beyond Beta.